src/caf/toolkit/cost_utils.py

```diff
@@ -262,14 +285,21 @@ def from_data(
         # Covert data into instance of this class
         df = pd.DataFrame(
             {
-                cls.min_col: bin_edges[:-1],
-                cls.max_col: bin_edges[1:],
-                cls.avg_col: (np.array(bin_edges[:-1]) + np.array(bin_edges[1:])) / 2,
-                cls.trips_col: distribution,
-                cls.weighted_avg_col: averages,
+                "min": bin_edges[:-1],
```

Comment:
Is it best practise not to access private class attributes from classmethods? I thought you were fine to access any private class functionality from any methods within the class?